cli/installer/install-azd.sh

```diff
@@ -171,7 +170,8 @@ architecture="$(get_architecture "$platform")"
 version="latest"
 dry_run=false
 skip_verify=false
-install_folder="/usr/local/bin"
+symlink_folder="/usr/local/bin"
+install_folder="/opt/microsoft/azd"
```

Comment:
Replaying @ellismg 's original [comment](https://github.com/Azure/azure-dev/pull/2032#issuecomment-1526424939) here:

Just wondering - what is pushing you towards the strategy of installing to `/opt/microsoft/azd` and then symlinking into `/usr/local/bin`? Some general MSFT wide guidance around putting the actual binaries under `/opt/microsoft` and then the symlink is to help us not have to add this folder to the `$PATH`?

Reply:
That explanation resonates with me. We definitely want a directory outside of `/usr/local/bin` to hold files.